integration_tests/tests/appointments/logCompliance.cy.ts

```diff
@@ -120,6 +120,10 @@ context('Log compliance', () => {
     cy.task('stubFindAppointment', { appointment: this.appointment })
     const page = LogCompliancePage.visit(this.appointment)
 
+    const form = appointmentOutcomeFormFactory.build()
+
+    cy.task('stubGetForm', form)
+
```

Comment:
What was happening before this was added out of interest?

Reply:
This is needed as we now require fetching the form for the log hours show page.